tools/flaky_test_monitor/common/config.go

```diff
@@ -0,0 +1,76 @@
+package common
+
+import (
+	"bufio"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// Properties used in property file
+const FailureThresholdPercent = "failures_threshold_percent"
+const FailuresSliceMax = "failures_slice_max"
+
+const DurationThresholdSeconds = "duration_threshold_seconds"
+const DurationSliceMax = "duration_slice_max"
+
+// const Duration
+
+type Config struct {
+	FailureThresholdPercent float32
+	FailuresSliceMax        int
+
+	DurationThresholdSeconds float32
+	DurationSliceMax         int
+}
+
+func ReadProperties(directory string) Config {
+	// looks for properties file with specific name
+	propertyFile := AddTrailingSlash(directory) + "flaky-test-monitor.properties"
```

Comment:
Could probably use https://pkg.go.dev/path/filepath#Join

Reply:
Good idea. I changed all occurrences of file path building to use it.

https://github.com/onflow/flow-go/pull/1793/commits/d59bce00cfd268deebe47b3dcb7d0e4776ef6abe